workshop-registration.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>WebDev Workshop Registration</title>
+    </head>
+    <body>
+        <h1 id="main-title">WebDev Workshop Registration</h1>
+        <p id="desc1">
+        Join our immersive workshop to sharpen your modern web development skills.<br>
+        Open to all, from beginners to experts—select sessions that match your passion and goals!
+        </p>
+        <form action="https://formspree.io/f/your-form-id" method="POST">
+            <fieldset>
+                <legend>Personal Information</legend>
+                <label for="full-name">Full Name</label><br>
+                <input type="text" id="full-name" name="fullName" required aria-label="Full Name"><br><br>
+                <label for="email">Email</label><br>
+                <input type="email" id="email" name="email" required aria-label="Email"><br><br>
+                <label for="city">City coming from?</label><br>
+                <select id="city" name="city" required aria-label="City coming from">
+                    <option value="">--Select City--</option>
+                    <option value="Mumbai">Mumbai</option>
+                    <option value="Bengaluru">Bengaluru</option>
+                    <option value="Delhi">Delhi</option>
+                    <option value="Chennai">Chennai</option>
+                    <option value="Pune">Pune</option>
+                    <option value="Hyderabad">Hyderabad</option>
+                    <option value="Kolkata">Kolkata</option>
+                    <option value="Ahmedabad">Ahmedabad</option>
+                    <option value="Jaipur">Jaipur</option>
+                    <option value="Lucknow">Lucknow</option>
+                </select><br><br>
+            </fieldset>
+            <fieldset>
+                <legend>Workshop Preferences</legend>
+                <span>Attendance type:</span><br>
```

Comment:
How will the screen reader users associate this text with the 2 radio inputs below?
That can only be done by wrapping the related inputs in a `fieldset` and using the `legend` tag to add this text as label of the fieldset

Reply:
Also, then we'll not having to use an unsemantic tag like `span`